src/components/teamFormation/createTeam.jsx

```diff
@@ -0,0 +1,113 @@
+import TextInput from "~/components/forms/textInput.jsx";
+import { useState } from "react";
+import ConfirmationModal from "~/components/confirmationModal.jsx";
+
+export default function CreateTeam() {
+  const [responseErrors, setResponseErrors] = useState("");
+  const [showModal, setShowModal] = useState(false);
+
+  async function submit(e) {
+    e.preventDefault();
+    closeModal();
+    const formData = new FormData(e.target);
+    const response = await fetch("/api/teams/create", {
+      method: "POST",
+      body: formData,
+    });
+
+    const data = await response.json();
+    if (!response.ok) {
+      setResponseErrors(data.message.errors);
+    } else {
+      window.location.href = "/";
+    }
+  }
+
+  function openModal() {
+    setShowModal(true);
+  }
+
+  function closeModal() {
+    setShowModal(false);
+  }
+
+  return (
+    <div className="mx-auto max-w-7xl px-4 sm:px-0 pb-24">
+      <div className="mx-auto max-w-3xl">
+        <form
+          encType="multipart/form-data"
```

Comment:
You can remove the attribute `encType`, since the user is not uploading a file.
By default it takes the value `application/x-www-form-urlencoded` and that's what you want 💪🏼